src/mrack/providers/openstack.py

```diff
@@ -76,14 +77,14 @@ def __init__(self):
             # https://docs.openstack.org/api-guide/compute/server_concepts.html
         }
 
-    async def init(self, image_names=None):
```

Comment:
There is a difference between image and image_name. Image is an object describing image in OpenStack. Image name is name of the image. And thus the init method uses `image_names` to make clear this distinction.